spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -67,14 +72,16 @@ public SparkRowLevelOperationsTestBase(String catalogName, String implementation
                 "default-namespace", "default"
             ),
             "orc",
-            true
+            true,
+            "none"
```

Comment:
Nit: I believe that there are constants for these, could you use those instead? In case the string value is ever updated etc =)